EzSystemsPrivacyCookieBundle.php

```diff
@@ -5,8 +5,13 @@
  */
 namespace EzSystems\PrivacyCookieBundle;
 
+use EzSystems\PrivacyCookieBundle\DependencyInjection\EzSystemsPrivacyCookieExtension;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 class EzSystemsPrivacyCookieBundle extends Bundle
 {
+    public function getContainerExtension()
+    {
+        return new EzSystemsPrivacyCookieExtension();
```

Comment:
Hah. I knew there was something in regards to the config key, sorry.

Maybe this is a hint that we should use `Ez` instead of `EzSystems` ;-)